src/locales/de/app.json

```
@@ -0,0 +1,132 @@
+{
+	"loading": "Lade...",
+	"download-ready": "Jetzt herunterladen!",
+	"navigation": {
+		"home": "Start",
+		"configuration": "Kalender erstellen",
+		"features": "Features",
+		"faq": "FAQ"
+	},
+	"language": {
+		"label": "Sprache",
+		"english": "Englisch",
+		"polish": "Polnisch",
+		"french": "Französisch",
+		"spanish": "Spanisch",
+		"german": "Deutsch"
+	},
+	"configuration": {
+		"selector": {
+			"label": "Start",
+			"template": {
+				"label": "Wähle eine Vorlage aus:",
+				"basic": "Einfach",
+				"advanced": "Fortschritten",
+				"blank": "Leer",
+				"minimalistic": "Minimalistisch"
+			},
+			"upload": {
+				"label": "Lade einen bestehendes ReCalendar PDF hoch um die Konfiguration zu übernehmen:",
+				"loading": "Lese Konfiguration aus dem PDF, bitte warten...",
+				"error": "Es konnte keine Konfiguration aus dem PDF übernommen werden - bist du sicher, dass das PDF mit dieser Website generiert worden ist?",
+				"success": "Konfiguration aus bestehendem PDF geladen - Willkommen zurück!"
+			}
+		},
+		"general": {
+			"label": "Allgemeine Optionen",
+			"year": "Jahr",
+			"starting-month": {
+				"label": "Start-Monat",
+				"description": "Das erste Monat, welches im Kalender enthalten seinsoll. Du kannst zB Oktober auswählen um einen Kalender für ein Semster, statt für ein Jahr zu erstellen."
```

I don't know if I can somehow extend on a previous suggestion, so I simply added this complete one, as there were more small issues in the last part of the text. I also would opt for bspw. as abbreviation of "beispielsweise"as it reads more fluently

```suggestion
				"description": "Der erste Monat, welcher im Kalender enthalten sein soll. Du kannst bspw. Oktober auswählen um einen Kalender für ein Semester statt für ein Jahr zu erstellen."
```